.firebaserc

```diff
@@ -1,5 +1,6 @@
 {
   "projects": {
-    "default": "staging-clockwork"
+    "default": "staging-clockwork",
```

Comment:
does this need to be changed to `staging-courseup`?

Reply:
No, because I messed up and I can't use `staging-courseup` for the time being we will use the current Firebase project for staging.